src/figuresClasses.ts

```diff
@@ -1,19 +1,114 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  color: string;
+
+  shape: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
```

Comment:
No need for so much newlines
```suggestion
  color: string;
  shape: string;
  a: number;
  b: number;
  c: number;
```

Reply:
Same for other classes